robotbase/src/main/java/com/systemmeltdown/robot/controls/GunnerControls.java

```diff
@@ -35,6 +46,12 @@ public GunnerControls(GunnerControlsBuilder builder) {
         m_leftTrigger = new AxisThresholdTrigger(builder.m_controller, Hand.kLeft, .1);
         m_yButton = new JoystickButton(builder.m_controller, XboxRaw.Y.value);
         m_xButton = new JoystickButton(builder.m_controller, XboxRaw.X.value);
+        m_aButton = new JoystickButton(builder.m_controller, XboxRaw.A.value);
```

Comment:
May want to implement a Climb Mode button on the shuffleboard, which keeps any of the commands being called until it's active. Otherwise, these binding location looks fine.